snapper.spec.in

```diff
@@ -188,7 +194,13 @@ Authors:
 /sbin/ldconfig
 
 %package -n libsnapper-devel
+%if 0%{?suse_version} > 1325
+Requires:       libboost_system-devel
+Requires:       libboost_thread-devel
+Requires:       libboost_test-devel
```

Comment:
You are correct. It seems it only needs 

    Snapper.h:#include <boost/noncopyable.hpp>

I'll update this so it only needs the boost headers then.